sdk/core/Microsoft.ClientModel.TestFramework/src/RecordedTests/RecordedTestBase.cs

```diff
@@ -89,6 +89,18 @@ public abstract class RecordedTestBase : ClientTestBase
     /// </value>
     public RecordedTestMode Mode { get; set; }
 
+    /// <summary>
+    /// Gets or sets a value indicating whether to remove all default sanitizers that are automatically turned on
+    /// in the test proxy. This can be helpful to improve performance in scenarios where libraries have very large
+    /// request bodies that are expensive to sanitize.
+    /// </summary>
+    /// <remarks>
+    /// This setting should be used with caution as it may lead to sensitive data being recorded. All sanitization,
+    /// will have to be manually added as a custom sanitizer if this is enabled. The only sanitizer left by default
+    /// is the Authorization header sanitizer.
+    /// </remarks>
+    public bool RemoveDefaultSanitizers { get; set; } = false;
```

Comment:
Should we make this a positive so that the default is more clear?   `EnableDefaultSanitizers`?